### ISSUE TITLE
[Components and pattern standards] The components of the navigation system (i.e. URL, breadcrumb, left nav) within your content or feature don't accurately and/or consistently represent the location of the page within the IA. (04.01.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#30436]
#### VFS team name
VANotify

#### VFS product name
Digital Notifications Terms & Conditions page on VA.gov

#### Point of Contact/Reviewers
Mikki Northuis (IA)

---

### Platform Issue
The components of the navigation system (i.e. URL, breadcrumb, left nav) within your content or feature don't accurately and/or consistently represent the location of the page within the IA.

### Issue Details
The breadcrumb on the preview page is not accurately showing the location of the page within the site.  The breadcrumb should be:  `Home > Privacy, policies, and legal information > Digital notifications terms and conditions`.  

### VA.gov Experience Standard
[Category Number 04, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.4 A

---

### Platform Recommendation
Correct the breadcrumb to <br> 1. Show "Home" as a clickable link that goes to the VA.gov home page.  2. Represent the Privacy policy page as part of the hierarchy.  3. Show the current page in the breadcrumb - the current page should be bolded and use the H1 of the page. 

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by mnorthuis** [comment]
This cannot be validated until the page is published. Removed launch-blocking lable, will validate once live.

**Comment by mnorthuis** [comment]
Breadcrumb on published page is not accurate. https://www.va.gov/digital-notifications-terms-and-conditions/

**Comment by RLHecht** [comment]
@mnorthuis This is now published with the breadcrumbs. I didn't enable the menu item, which means a left nav won't show. If there should be one, let me know and I will set that up.


<img width="841" alt="Screen Shot 2021-09-28 at 4 00 44 PM" src="https://user-images.githubusercontent.com/48728214/135157356-69fd3fea-cb46-489f-b404-b37baaa8f170.png">
